### ISSUE TITLE
Weekly Scrum [2020-03-25]

### ISSUE THREAD

**Issue by JoseFrancoJimenez** [JoseFrancoJimenez/miniShopify#108]
1- what have I done this week? (with link(s) to the relevant GitHub Issue(s))
2- what will I do next week? (with link(s) to the relevant GitHub Issue(s))
3- what is holding me back? (if applicable)

**Comment by JoseFrancoJimenez** [comment]
1- what have I done this week? (with link(s) to the relevant GitHub Issue(s))
  TA reported some bugs, I created an issue for each of them, assigned to me, and worked on it (see issues: https://github.com/JoseFrancoJimenez/miniShopify/issues/103, https://github.com/JoseFrancoJimenez/miniShopify/issues/104, https://github.com/JoseFrancoJimenez/miniShopify/issues/105, https://github.com/JoseFrancoJimenez/miniShopify/issues/106, https://github.com/JoseFrancoJimenez/miniShopify/issues/107 )

2- what will I do next week? (with link(s) to the relevant GitHub Issue(s))

3- what is holding me back? (if applicable)

**Comment by Aryanrt** [comment]
1- what have I done this week? (with link(s) to the relevant GitHub Issue(s))
I worked in a separate project to do the project in JHipster. I was able to add Shop and Product entities. the repo can be found at: 
https://github.com/Aryanrt/anotherone   
deployed at:
https://team24-anotherone.herokuapp.com/

2- what will I do next week? (with link(s) to the relevant GitHub Issue(s))
I will try to finish the project for milestone 3. There are a lot of little bugs and no error handling for the forms yet

3- what is holding me back? (if applicable)
Nothing, I am working my behind off
